custom_components/tuya_local/devices/warmehaus_AFD02TJ_thermostat_c.yaml

```diff
@@ -0,0 +1,76 @@
+name: WarmeHaus Thermostatic Towel Rail Electric Heating Element
+products:
+  - id: fgzmbjxhgkuwgrav
+    manufacturer: WarmeHaus
+    model: AFD-02-TJ
+entities:
+  - entity: climate
+    dps:
+      - id: 1
+        type: boolean
+        name: hvac_mode
+        mapping:
+          - dps_val: false
+            value: 'off'
+          - dps_val: true
+            constraint: mode
+            conditions:
+              - dps_val: cold
+                value: cool
+              - dps_val: hot
+                value: heat
+              - dps_val: eco
+                value: heat
+      - id: 1
+        type: boolean
+        name: hvac_action
+        mapping:
+          - dps_val: false
+            value: 'off'
+          - dps_val: true
+            constraint: mode
+            conditions:
+              - dps_val: cold
+                value: cooling
+              - dps_val: ['hot', 'eco', 'auto']
+                value: heating
+      - id: 2
+        type: string
+        name: mode
+        hidden: true
+      - id: 16
+        type: integer
+        name: temperature
+        range:
+          min: 300
+          max: 700
+        mapping:
+          - scale: 10
+            step: 10
+      - id: 24
+        type: integer
+        name: current_temperature
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 40
+        optional: true
+        type: boolean
+        name: lock
+  - entity: number
+    name: Temperature calibration
+    category: config
+    icon: 'mdi:thermometer-check'
+    dps:
+      - id: 27
+        type: integer
+        optional: true
+        name: value
+        unit: C

```

Comment:
Calibration uses relative temperatures, so needs to have ° as the unit to avoid strange conversion issues if the user's HA setting does not match the device.